templates/_helpers.tpl

```diff
@@ -39,7 +47,9 @@ Create the name of the stackstorm-ha service account to use
 {{- include $template (dict "Chart" (dict "Name" (last $subchart)) "Values" $values "Release" $dot.Release "Capabilities" $dot.Capabilities) }}
 {{- end }}
 
-# Generate comma-separated list of nodes for MongoDB-HA connection string, based on number of replicas and service name
+{{/*
+Generate comma-separated list of nodes for MongoDB-HA connection string, based on number of replicas and service name
+*/}}
 {{- define "mongodb-nodes" -}}
```

Comment:
I didn't prefix this, because I'm not sure if we're overriding a subchart template helper.